test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -3035,12 +2984,12 @@ static public void testUnresolvedPutFieldUse() throws Throwable {
 	static public void testIdentityObjectOnValueType() throws Throwable {
```

Comment:
Leave these tests as is, same as below. We will need more changes to get this working with the RI. These changes will be done in other PRs